### PR TITLE
[JR-139] Fix footer, unbounce channel search

### PR DESCRIPTION
**Pull request by E1379**

### CODE REVIEW

File: src/views/MediaChannels/components/Sidebar.tsx [hunk 1]
```diff
@@ -27,21 +28,50 @@ export const Sitebar: React.FC<SitebarProps> = ({
 }) => {
   const classes = useStyles();
   const intl = useIntl();
+  const [searchText, setSearchText] = useState(criteria.name);
```
Comment (marianmazarovici) [inline src/views/MediaChannels/components/Sidebar.tsx:31]: what's this supposed to do?

Comment (E1379) [reply src/views/MediaChannels/components/Sidebar.tsx:31]: This creates a state variable, initialized with the criteria.name.

Comment (E1379) [reply src/views/MediaChannels/components/Sidebar.tsx:31]: And this is needed to debounce the change event, so not every keystroke ends up with a query to pkb

Comment (marianmazarovici) [reply src/views/MediaChannels/components/Sidebar.tsx:31]: sorry, I mean the whole commit (except fixes to footer menu)

Comment (E1379) [reply src/views/MediaChannels/components/Sidebar.tsx:31]: > this is needed to debounce the change event, so not every keystroke ends up with a query to pkb

**Review by marianmazarovici — APPROVED** [review]

**Pull request merged**